extensions/toolbox/src/script/string/camelCase.ts

```diff
@@ -0,0 +1,9 @@
+import { Script } from "../type";
+import * as _ from "lodash";
+
+export const camelCase: Script = {
+  info: { title: "Camel Case", desc: "Converts Your Text To Camel Case", type: "all", example: "ray cast" },
```

Comment:
`Converts Your Text To Camel Case` should be `Convert your text into Camel Case`.

Reply:
Thank you. I fixed it.